config/config_test.go

```diff
@@ -52,7 +52,7 @@ func (s *ConfigTestSuite) TestTimezone() {
 
 	// avoid Exit(255) on glog.Fatal
 	monkey.Patch(glog.Fatal, func(a ...interface{}) {
-		s.Contains(a[0], "cannot find nolnexistent in zip file")
+		s.Contains(a[0], "unknown time zone nolnexistent")
```

Comment:
```go
s.Contains(a[0], "cannot find nolnexistent in zip file") || 
  s.Contains(a[0], "unknown time zone nolnexistent")
```

Reply:
just tried this and this doesn't appear to work. I'm tempted to get rid of this entirely. I dont' think it is worthwhile to test that the program will Fatal exit if an invalid timezone is provided

Reply:
I agree and just removed those lines from this commit